package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "express-typescript-starter",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "node dist/index.js",
+    "build": "tsup src/*",
+    "watch": "tsup src/* --watch",
+    "dev": "nodemon dist/index.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@thirdweb-dev/sdk": "^3.6.9",
+    "cors": "^2.8.5",
+    "dotenv": "^16.0.2",
```

Comment:
are these strictly required to run the thing?

Reply:
I think I can remove cors, my bad but dotenv is required yea.

Reply:
dotenv is required for which part? private key? I believe we now have a way to properly grab a private key without .env files? @adam-maj please confirm

Reply:
ooh didn't know about that

Reply:
We have several ways to grab private key from things like AWS KMS/AWS Secrets Manager, but that probably isn't ideal for this type of setup since it's quite opinionated

Reply:
So probably best to leave dotenv for now